index.bs

```diff
@@ -47,66 +51,88 @@ dictionary CSSStyleSheetInit {
+		4. If the {{CSSStyleSheetInit/alternate}} attribute of <var>options</var> is true,
 			set <var>sheet’s</var> alternate flag.
-		4. If the {{CSSStyleSheetInit/disabled}} attribute of <var>options</var> is true,
+		5. If the {{CSSStyleSheetInit/disabled}} attribute of <var>options</var> is true,
 			set <var>sheet’s</var> disabled flag.
-		5. <a spec=css-syntax-3>Parse a stylesheet</a> from {{text}}.
+		6. Return <var>sheet</var>.
+	</dd>
+	<dt><dfn  for=CSSStyleSheet lt="constructed flag">constructed flag</dfn></dt>
+	<dd>
+		Specified when created. Either set or unset. Unset by default.
+		Signifies whether this stylesheet is made via constructor or not, so must be set only for stylesheets that are constructed using the {{CSSStyleSheet()}} or {{CSSStyleSheet(options)}} function.
+	</dd>
+	<dt><dfn  for=CSSStyleSheet lt="disallow modification flag">disallow modification flag</dfn></dt>
+	<dd>
+		Either set or unset. Unset by default. If set, modification to the stylesheet's rules are not allowed.
+	</dd>
+	<dt><dfn  for=CSSStyleSheet lt="constructor document">constructor document</dfn></dt>
+	<dd>
+		Specified when created. The {{Document}} where the CSSStyleSheet is originally constructed on. Null by default.
+	</dd>
+	<dt><dfn  for=CSSStyleSheet lt="set of adopter documents">set of adopter documents</dfn></dt>
+	<dd>
+		A set of {{Document}}s on which the stylesheet is adopted (e.g. there is at least one {{DocumentOrShadowRoot}} within that {{Document}}'s tree on which the sheet is part of the {{DocumentOrShadowRoot}}'s {{DocumentOrShadowRoot/adoptedStyleSheets}}).
+	</dd>
+</dl>
+
+Modifying Constructed Stylesheets {#modifying-constructed-stylesheets}
+=============================
+
+After construction, constructed stylesheets can be modified using rule modification methods like <a spec="cssom-1" for="CSSStyleSheet">insertRule(rule)</a> or <a spec="cssom-1" for="CSSStyleSheet">deleteRule(rule)</a>, or {{replace(text)}} and {{replaceSync(text)}} if the sheet's [=disallow modification flag=] is not set. If those methods are called when the sheet's [=disallow modification flag=] is set, or <a spec="cssom-1" for="CSSStyleSheet">insertRule(rule)</a> is used to add an <a spec=css-cascade-4>@import</a> rule, a "{{NotAllowedError}}" {{DOMException}} must be thrown. 
+
+<dl>
+    <dt><dfn method for=CSSStyleSheet lt="replace(text)">replace(text)</dfn></dt>
+    <dd>
+		1. Let <var>sheet</var> be the stylesheet on which this function is called on.
+		2. If <var>sheet’s</var> [=constructed flag=] is not set, or <var>sheet's</var> [=disallow modification flag=] is set, throw a "{{NotAllowedError}}" {{DOMException}}.
+		3. Set <var>sheet’s</var> CSS rules to an empty list, and set <var>sheet's</var> [=disallow modification flag=].
+		4. <a spec=css-syntax-3>Parse a stylesheet</a> from {{text}}.
 			If it returned a list of rules,
 			assign the list as <var>sheet’s</var> CSS rules;
 			otherwise,
 			set <var>sheet’s</var> CSS rules to an empty list.
```

Comment:
Preexisting editorial issue: every time you reference a "field" of the stylesheet concept, you should be cross-referencing it. E.g. `[=CSSStyleSheet/CSS rules=]`. But, I guess these are currently not exported, so that won't work... something to fix when we do the eventual integration into CSSOM.

Reply:
Added cross references.